src/io/flutter/run/daemon/FlutterApp.java

```diff
@@ -143,17 +146,24 @@ public static FlutterApp start(@NotNull ExecutionEnvironment env,
 
     // Send analytics for the start and stop events.
     FlutterInitializer.sendAnalyticsAction(analyticsStart);
+
+    final DaemonApi api = new DaemonApi(process);
+    final FlutterApp app = new FlutterApp(project, module, mode, device.deviceId(), process, env, api);
+
     process.addProcessListener(new ProcessAdapter() {
       @Override
       public void processTerminated(ProcessEvent event) {
         LOG.info(analyticsStop + " " + project.getName() + " (" + mode.mode() + ")");
         FlutterInitializer.sendAnalyticsAction(analyticsStop);
+
+        // Send analytics about whether this session used the reload workflow, the restart workflow, or neither.
+        final String workflowType = app.reloadCount > 0 ? "reload" : (app.restartCount > 0 ? "restart" : "none");
```

Comment:
What if the user does both reload & restart? Either we don't care, and it will be recorded as a reload. Or we do care, and we need a fourth identifier.

Reply:
The way we're measuring _is_ a bit coarse. What we really want to know is the % of people using hot reload in their workflow (and even more specifically, the % that aren't, and then try and improve that). This counts a reload+restart in one session as a 'reload' workflow, but I think that's fine from the POV of what we really want to measure.